spec/models/instance_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe Instance, type: :model do
+RSpec.describe Instance do
```

Comment:
Hmm, I don't know this. It's only for model test ? or feature/controller test as well ?